subschemas/substrate-chain/src/typeDefs/unionTypes.ts

```diff
@@ -0,0 +1,3 @@
+export default /* GraphQL */ `
+  union Proposal = TreasuryProposal | MotionProposal | DemocracyProposal | DemocracyReferendum
```

Comment:
You've created this union type but you never use it.
Weren't you supposed to replace these types with the new union?

Reply:
Actually going to remove it. Eventually ended up not needing it in the client.